### ISSUE TITLE
particles.js blocked by adblock/ublock lists

### ISSUE THREAD

**Issue by sebnapi** [VincentGarreau/particles.js#351]
Unfortunatly some ad block plugins block the plugin now https://easylist-msie.adblockplus.org/fanboy-annoyance.tpl

Which is merged into "Fanboy+Easylist-Merged Ultimate List​"

Just in case somebody is wondering.

**Comment by benzmuircroft** [comment]
I renamed particles.js and particles.json with random script names and they are no longer adblocked
